src/test/ui/macros/issue-62894.rs

```diff
@@ -0,0 +1,7 @@
+// Regression test for #62894, shouldn't crash.
+// error-pattern: this file contains an unclosed delimiter
+// error-pattern: expected one of `(`, `[`, or `{`, found keyword `fn`
+
+fn f() { assert_eq!(f(), (), assert_eq!(assert_eq!
```

Comment:
Let's move the test into `ui/parser/` as it doesn't seem specific to macro expansion?

Reply:
Moved.